GitUI/Translation/English.Plugins.xlf

```diff
@@ -1175,66 +893,6 @@ Do you want to reset the configured path?</source>
       </trans-unit>
     </body>
   </file>
-  <file datatype="plaintext" original="JiraCommitHintPlugin" source-language="en">
```

Comment:
I hate error-hiding!

![image](https://user-images.githubusercontent.com/36601201/210186264-b6424ef6-1e92-474f-80e6-6cb75d324417.png)

```
  ERROR getting types of d:\Build\gitextensions3_dev\artifacts\Debug\bin\GitExtensions\net6.0-windows\Plugins\JiraCommitHintPlugin\GitExtensions.Plugins.JiraCommitHintPlugin.dll:
  System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
  Could not load file or assembly 'Atlassian.Jira, Version=12.4.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.
     at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
     at System.Reflection.Assembly.GetTypes()
     at ResourceManager.Xliff.TranslationUtil.GetLoadableTypes(Assembly assembly) in d:\Build\gitextensions3_dev\ResourceManager\Xliff\TranslationUtil.cs:line 368
  System.IO.FileNotFoundException: Could not load file or assembly 'Atlassian.Jira, Version=12.4.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.
  File name: 'Atlassian.Jira, Version=12.4.0.0, Culture=neutral, PublicKeyToken=null'
  Processing the following types anyway:
  Microsoft.CodeAnalysis.EmbeddedAttribute
  System.Runtime.CompilerServices.NullableAttribute
  System.Runtime.CompilerServices.NullableContextAttribute
  Microsoft.Validates
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin
  GitExtensions.Plugins.JiraCommitHintPlugin.Properties.Resources
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+JiraTaskDTO
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<<Execute>b__22_0>d
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass25_0
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass30_0
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass30_1
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass32_0
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<GetSettings>d__23
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass25_0+<<btnPreviewClick>b__0>d
  GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin+<>c__DisplayClass30_0+<<gitUiCommands_PreCommit>b__0>d
  ERROR instantiating type GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin:
  System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
   ---> System.IO.FileNotFoundException: Could not load file or assembly 'Atlassian.Jira, Version=12.4.0.0, Culture=neutral, PublicKeyToken=null'. Das System kann die angegebene Datei nicht finden.
  File name: 'Atlassian.Jira, Version=12.4.0.0, Culture=neutral, PublicKeyToken=null'
     at GitExtensions.Plugins.JiraCommitHintPlugin.JiraCommitHintPlugin..ctor()
     at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
     --- End of inner exception stack trace ---
     at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
     at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
     at System.Activator.CreateInstance(Type type, Boolean nonPublic)
     at ResourceManager.Xliff.TranslationUtil.CreateInstanceOfClass(Type type) in d:\Build\gitextensions3_dev\ResourceManager\Xliff\TranslationUtil.cs:line 350
     at TranslationApp.TranslationHelpers.LoadNeutralItems() in d:\Build\gitextensions3_dev\TranslationApp\TranslationHelpers.cs:line 26
```

Reply:
Thank you for the update.